static/app/views/releases/detail/overview/issues.tsx

```diff
@@ -141,7 +199,64 @@ class Issues extends Component<Props, State> {
     }
   }
 
+  async fetchIssuesCount() {
+    const {api, version} = this.props;
+    const endpoint = this.getIssueCountEndpoint();
+
+    try {
+      const response = await api.requestPromise(endpoint);
+      this.setState({
+        count: {
+          release: response[`release:${version}`],
+          firstRelease: response[`first-release:${version}`],
+          resolved: response['is:resolved'],
+          unhandled: response['error.handled:0'],
+        },
+      });
+    } catch {
+      // do nothing
+    }
+  }
+
+  getIssueCountEndpoint() {
+    const {organization, version} = this.props;
+    const path = `/organizations/${organization.slug}/issues-count/`;
+    const params = [
+      `first-release:${version}`,
+      `release:${version}`,
+      'is:resolved',
```

Comment:
I'm afraid `issues-count` endpoint won't return issues resolved by a release. There's another endpoint for that. (you can inspect network tab on release details when you change the dropdown)